src/Form/EntryForm.php

```diff
@@ -154,6 +163,68 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
   }
 
   /**
+   * Checks if user is allowed to continue with plugin action.
+   *
+   * @param string $window
+   * @return bool
+   */
+  public function floodIsAllowed($window = '') {
+    if (method_exists($this->tfaBasePlugin, 'floodIsAllowed')) {
+      return $this->tfaBasePlugin->floodIsAllowed($window);
+    }
+    return TRUE;
+  }
+
+  /**
+   * Check if flood has been hit.
+   * @param Tfa $tfa
+   * @return bool
+   */
+   // Argument of this function has to be changed , depending upon getContext function
+  public function _tfa_hit_flood($tfa) {
```

Comment:
Is this a public function? The underscore in the function name suggests its likely private and you do not need the underscore at the beginning of the function name.

Proper class naming conventions for methods would be tfaHitFloor